NEWS

```diff
@@ -1,5 +1,20 @@
 systemd System and Service Manager
 
+CHANGES WITH 236:
+
+	* The modprobe.d drop-in introduced in v235 has been extended to also
+	  set the dummy module option numdummies=0, resolving issues with the
+	  kernel creating dummy0.
+
+        Contributions from: bleep_blop, Christian Hesse, Daniel Rusek, Florian
```

Comment:
I think it's better to skip the contributor list now, and just create it at the end.